_includes/components/svg.html

```diff
@@ -0,0 +1,12 @@
+{% comment %}
+svg
+- filename {String}
+- class {String}
+- width {Number} width in pixels
+{% endcomment %}
+
+{% assign iconPath = "svg/" | append: include.filename | append: ".svg" %}
+
+<div class="{{ "svg " | append: include.class }}" {% if include.width %}style="{{ "width: " | append: include.width | append: ";"}}"{% endif %} >
+  {% include {{ iconPath }} %}
+</div>
```

Comment:
Wound up having to wrap the svg in a div since I'm not sure I can edit the svg attributes directly. Since we're not using Tailwind, I allowed an arbitrary width override with a default width set by the class.